JPS_Ontology/ontology/ontoems/OntoEMSExtension.csv

```diff
@@ -0,0 +1,107 @@
+AtmosphericPressure,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Pressure,IS-A,,,,The pressure exerted by the atmospheric column of air above the point in question (adjusted to mean sea level).,https://www.theworldavatar.com/kg/ontoems/,Atmospheric pressure
+AtmosphericPressure,Class,http://purl.org/iot/vocab/m3-lite#AtmosphericPressure,EQUIVALENT-TO,,,,The pressure exerted by the atmospheric column of air above the point in question (adjusted to mean sea level).,https://www.theworldavatar.com/kg/ontoems/,Atmospheric pressure
+AtmosphericPressure,Class,https://www.auto.tuwien.ac.at/downloads/thinkhome/ontology/WeatherOntology.owl#AtmosphericPressure,EQUIVALENT-TO,,,,The pressure exerted by the atmospheric column of air above the point in question (adjusted to mean sea level).,https://www.theworldavatar.com/kg/ontoems/,Atmospheric pressure
+RelativeHumidity,Class,http://purl.org/iot/vocab/m3-lite#RelativeHumidity,EQUIVALENT-TO,,,,Relative humidity of the outside air.,https://www.theworldavatar.com/kg/ontoems/,Relative humidity
+RelativeHumidity,Class,https://www.auto.tuwien.ac.at/downloads/thinkhome/ontology/WeatherOntology.owl#Humidity,EQUIVALENT-TO,,,,Relative humidity of the outside air.,https://www.theworldavatar.com/kg/ontoems/,Relative humidity
+RelativeHumidity,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/RelativeHumidity,EQUIVALENT-TO,,,,Relative humidity of the outside air.,https://www.theworldavatar.com/kg/ontoems/,Relative humidity
+Visibility,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Distance,IS-A,,,,"Prevailing horizontal distance at which an object can be clearly discerned by a certified weather observer, under current weather conditions.",https://www.theworldavatar.com/kg/ontoems/,Visibility
+UVIndex,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Number,IS-A,,,,The strength of the sun's ultraviolet (UV) radiation expressed as index as developed by the World Health Organization.,https://www.theworldavatar.com/kg/ontoems/,UV index
+PrecipitationProbability,Class,http://www.ontology-of-units-of-measure.org/resource/om-2/Number,IS-A,,,,The likelihood of occurrence of precipitation.,https://www.theworldavatar.com/kg/ontoems/,Precipitation probability
+PrecipitationProbability,Class,http://semanticscience.org/resource/SIO_000638,IS-A,,,,The likelihood of occurrence of precipitation.,https://www.theworldavatar.com/kg/ontoems/,Precipitation probability
+AirPollutantConcentration,Class,https://www.theworldavatar.com/kg/ontouom/Concentration,IS-A,,,,The concentration of a specific air pollutant as amount of material per unit volume of air (e.g. mass concentration).,https://www.theworldavatar.com/kg/ontoems/,Air pollutant concentration
+http://www.ontology-of-units-of-measure.org/resource/om-2/VolumeFraction,Class,https://www.theworldavatar.com/kg/ontouom/Concentration,IS-A,,,,,https://www.theworldavatar.com/kg/ontoems/,Volume fraction
```

Comment:
Please note that volume fraction is _not_ a concentration (see e.g. https://en.wikipedia.org/wiki/Concentration), so please delete this. Any other statements you'd like to make about volume fraction should be in OM, not here.

Reply:
While it may well not be relevant here, I'd rather not do things that are obviously wrong, if it can be avoided... Shame that OM doesn't get it right, either. I'm not suggesting that we fix problems in OM that we haven't created ourselves, but at least we shouldn't add more things that are wrong or make it worse...